network/data_link/wifi.c

```diff
@@ -59,7 +59,7 @@
 /******************************************************************************/
```

Comment:
typo in commit message: Raname -> Rename

Reply:
solved